src/main/java/ledger/user_interface/ui_controllers/window/ImportTransactionsPopupController.java

```diff
@@ -100,24 +112,42 @@ private void handleReturn(ImportController.ImportFailures importFailures) {
         for (Transaction fail : importFailures.failedTransactions) {
             // Todo: Do we even want to show the user?
         }
+        genericTransactionsWindow(importFailures.ignoredTransactions,
```

Comment:
I was going off this quote
"I think it would be better if the ignore worked a lot more how duplicates are detected. That way can tell the user what was ignored instead of it just being magically handled by the backend." -cjm721